lib/common/Memory/Chakra.Common.Memory.vcxproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Was this a BOM change? If it doesn't hurt I think you should keep the BOM since some of our tools complain or will try to change it back.